lib/pliny/helpers/serialize.rb

```diff
@@ -1,6 +1,6 @@
 module Pliny::Helpers
   module Serialize
-    def self.included(base)
+    def self.prepended(base)
```

Comment:
Could we just have _both_? Then we wouldn't have to require sinatra >=2.1.

Reply:
We could for sure, I'd like to keep the maintenance cost low and cut old versions quickly. This change in Pliny is the only reason why we can't use Sinatra 2.1 there shouldn't be any other blockers for your project to move to Sinatra 2.1 with this new version of Pliny. if you don't want to move to Sinatra 2.1 you can stay on the current Pliny version without missing anything. We are not shipping new features in Pliny, it's basically in maintenance mode only at this point.